template/deploy/helm/[[product]]-operator/templates/deployment.yaml

```diff
@@ -34,6 +34,11 @@ spec:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
         - name: {{ include "operator.appname" . }}
+{[% if operator.product_string in ['zookeeper']%}]
+# Reworked ("STFU") operators require a run argument
+          args:
+            - run
+{[% endif %}]
```

Comment:
* deployment.yaml is not a Jinja2 template.
* The `run` argument could also be added to the Dockerfile. The Docker container would then perform the "default" operation when started and not just complain about missing arguments.
* I would not make the argument conditional. The generated pull request should then be merged when the rework is done. Admittedly, updates on operators which are not reworked yet would not be possible.

Reply:
300fdd60d75d0dfade47b4f81ec6866ab0b6721a